producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigBTwo.java

```diff
@@ -0,0 +1,266 @@
+import uk.gov.moj.sdt.producers.comx.utils.BulkFeedbackFactory;
+
+@Configuration
+@EnableAutoConfiguration
+public class BulkFeedbackConfigBTwo {
+
+    @Bean("createIndividualRequestB0000002")
+    public void invokingBean() {
+        MethodInvokingFactoryBean methodInvokingFactoryBean = new MethodInvokingFactoryBean();
+        methodInvokingFactoryBean.setTargetObject(bulkFeedbackFactory());
+        methodInvokingFactoryBean.setTargetMethod("createIndividualRequests");
+
+        methodInvokingFactoryBean.setArguments(Lists.newArrayList(
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B1a",
+                "mcolClaim",
+                "Rejected",
+                "DUP_CUST_REQID",
+                "Duplicate User Request Identifier submitted USER_REQUEST_ID_B1a.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B1b",
+                "mcolJudgment",
+                "Rejected",
+                "DUPLD_CUST_REQID",
+                "Unique Request Identifier has been specified more than once within the originating Bulk Request.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B1b",
+                "mcolJudgment",
+                "Rejected",
+                "DUPLD_CUST_REQID",
+                "Unique Request Identifier has been specified more than once within the originating Bulk Request.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B2",
+                "mcolClaim",
+                "Rejected",
+                "8",
+                "First defendant's postcode is not in England or Wales.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B3",
+                "mcolJudgment",
+                "Rejected",
+                "75",
+                "Rejected by CCBC - rejection number 6.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B4",
+                "mcolClaimStatusUpdate",
+                "Accepted",
+                "",
+                "",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B5",
+                "mcolJudgment",
+                "Rejected",
+                "24",
+                "This judgment request is invalid on the referenced claim.",
+                ""
+            ),
+            Lists.newArrayList(
+                "USER_REQUEST_ID_B6",
+                "mcolWarrant",
+                "Accepted",
+                "",
+                "",
+                """
+                      <![CDATA[
+                          <ind:mcolResponseDetail xmlns:ind="http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema" xmlns:mresp="http://ws.sdt.moj.gov.uk/2013/mcol/ResponseDetailSchema">
+                          	  <mresp:issueDate>2014-01-23</mresp:issueDate>
```

Comment:
Line has one tab character in it two spaces before the `<m:resp>` tag, i.e.

`                          [tab]  <mresp:...`

This causes a SonarLint "Remove the non-escaped \u0009 character from this literal." error to be flagged in IntelliJ.  Replace it with spaces to resolve the SonarLint error.